index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desktopview</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Gilda+Display&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&family=Urbanist:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="desktop-full">
+        <div class="navbar">
+        <div class="Travel">Travel</div>
+            <img src="img/Frame 1664.svg" alt="">
+        </div>
+        <div class="header">
+        <div class="overlap-group">
+        <div class="blurry"></div>
+        <div class="circle-background"><div class="ellipse"></div></div>
+        <p class="travel-around-the">
+        <span class="text-wrapper">Travel</span> <span class="span"> around the world</span>
+        </p>
+        <img class="image" src="img/Image (4).png" />
+        </div>
+    </section>
```

Comment:
Comment on Indentation: The section was correctly implemented but the children tags within the section tag should be indented. E.g 
<section>
    <div>Text 1</div>
    <div>Text 1</div>
</section>

Go through your ...